sequencing/vcf_slice/Readme.md

```diff
@@ -1,21 +1,32 @@
-<!-- dx-header -->
 # Slice VCF File(s) (DNAnexus Platform App)
 
-Return a small section of a VCF file (similar to tabix).  For large output, many small regions, or subsetting samples, use subset_vcf instead.
+Return a small section of a VCF file (similar to tabix).  For large output, 
+many small regions, or subsetting samples, use the app `select_variants` instead.
 
-This is the source code for an app that runs on the DNAnexus Platform.
-For more information about how to run or modify it, see
-https://wiki.dnanexus.com/.
-<!-- /dx-header -->
+## App Inputs
 
+This app requires 1 or more variant files in VCF format to be filtered to the
+target region and a corresponding Tabix index file as well as a region file
```

Comment:
Specify that it's (`*.vcf.gz.tbi`) and whatever the format for the region file is